app/assets/stylesheets/nollning.scss

```diff
@@ -0,0 +1,33 @@
+$top: "theme/top.svg";
+$pillars: "theme/pillars.svg";
+$stairs: "theme/stairs.svg";
+.roman-top {
+  border: 90px solid transparent;
+  padding-top: 25px;
```

Comment:
Properties should be ordered border, -o-border-image, -webkit-border-image, border-image, padding-top